wondrous-app/components/Common/TaskViewModal/taskViewModalFields.tsx

```diff
@@ -299,10 +279,7 @@ export function AssigneeField({
                 variables: {
                   taskId: fetchedTask?.id,
                 },
-                onCompleted: (data) => {
-                  const task = data?.removeTaskAssignee;
-                  updateBoard({ task, setFetchedTask, boardColumns });
```

Comment:
Will this update the board after updating in the card tho?

Reply:
Actually seems like it does!

Reply:
Yes, according to the docs

> All subscribers to the Apollo Client cache (including all active queries) see this change and update your application's UI accordingly.
https://www.apollographql.com/docs/react/caching/cache-interaction/#writefragment